tests/integration/api/v1/test_common_serializers.py

```diff
@@ -0,0 +1,134 @@
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+import json
+
+import pytest
+
+
+def test_literature_recids_serializer(api_client):
+    response = api_client.get(
+        '/literature/?q=title%20collider',
+        headers={'Accept': 'application/vnd+inspire.ids+json'}
+    )
+
+    assert response.status_code == 200
+
+    response_json = json.loads(response.data)
+
+    assert response_json['hits']['total'] == 2
+
+    expected_recids = {1373790, 701585}
+    response_recids = set([response_json['hits']['recids'][idx] for idx in range(response_json['hits']['total'])])
+
+    assert response_recids == expected_recids
+
+
+def test_authors_recids_serializer(api_client):
+    response = api_client.get(
+        '/authors/?q=arxiv_categories:"hep-ph"',
+        headers={'Accept': 'application/vnd+inspire.ids+json'}
+    )
+
+    assert response.status_code == 200
+
+    response_json = json.loads(response.data)
+
+    assert response_json['hits']['total'] == 4
+
+    expected_recids = {1057204, 984519, 1073117, 1060891}
+    response_recids = set([response_json['hits']['recids'][idx] for idx in range(response_json['hits']['total'])])
+
+    assert response_recids == expected_recids
+
+
+def test_conferences_recids_serializer(api_client):
+    response = api_client.get(
+        '/conferences/',
+        headers={'Accept': 'application/vnd+inspire.ids+json'}
+    )
+
+    assert response.status_code == 200
+
+    response_json = json.loads(response.data)
+
+    assert response_json['hits']['total'] == 1
+
+    assert response_json['hits']['recids'][0] == 972464
+
+
+def test_institutions_recids_serializer(api_client):
+    response = api_client.get(
+        '/institutions/',
+        headers={'Accept': 'application/vnd+inspire.ids+json'}
+    )
+
+    assert response.status_code == 200
+
+    response_json = json.loads(response.data)
+
+    assert response_json['hits']['total'] == 1
+
+    assert response_json['hits']['recids'][0] == 902725
+
+
+def test_experiments_recids_serializer(api_client):
+    response = api_client.get(
+        '/experiments/',
+        headers={'Accept': 'application/vnd+inspire.ids+json'}
+    )
+
+    assert response.status_code == 200
+
+    response_json = json.loads(response.data)
+
+    assert response_json['hits']['total'] == 1
+
+    assert response_json['hits']['recids'][0] == 1108642
+
+
+def test_journals_recids_serializer(api_client):
+    response = api_client.get(
+        '/journals/',
+        headers={'Accept': 'application/vnd+inspire.ids+json'}
+    )
+
+    assert response.status_code == 200
+
+    response_json = json.loads(response.data)
+
+    expected_recids = {1214516, 1213103}
+    response_recids = set([response_json['hits']['recids'][idx] for idx in range(response_json['hits']['total'])])
+
+    assert response_recids == expected_recids
+
+
+@pytest.mark.xfail(reason='Due to jobs mapping missing earliest_date field to sort on.')
```

Comment:
Please add a commit to fix this, it sounds quite straightforward!

Reply:
Will do now!